addons/survey/tests/test_survey_ui_feedback.py

```diff
@@ -226,22 +229,37 @@ def test_04_public_survey_with_triggers(self):
                     ],
                     'is_conditional': True,
                     'constr_mandatory': True,
-                }),
+                }), (0, 0, {
+                    'title': 'Q4',
+                    'sequence': 4,
+                    'question_type': 'numerical_box',
+                    'is_conditional': True,
+                    'constr_mandatory': True,
+                }), (0, 0, {
+                    'title': 'Q5',
+                    'sequence': 5,
+                    'question_type': 'numerical_box',
+                    'is_conditional': True,
+                })
             ]
         })
 
-        q1 = survey_with_triggers.question_ids.filtered(lambda q: q.title == 'Q1')
-        q1_a1 = q1.suggested_answer_ids.filtered(lambda a: a.value == 'Answer 1')
-        q2 = survey_with_triggers.question_ids.filtered(lambda q: q.title == 'Q2')
-        q2_a1 = q2.suggested_answer_ids.filtered(lambda a: a.value == 'Answer 1')
-        q3 = survey_with_triggers.question_ids.filtered(lambda q: q.title == 'Q3')
+        q1, q2, q3, q4, q5 = survey_with_triggers.question_and_page_ids
+        q1_a1, q1_a2 = q1.suggested_answer_ids[0:2]
```

Comment:
This is enough
```suggestion
        q1_a1, q1_a2 = q1.suggested_answer_ids[:2]
```
To be complete, it could also be
```suggestion
        q1_a1, q1_a2, *__ = q1.suggested_answer_ids
```
or even, as there are only 3 answers, perhaps the most readable:
```suggestion
        q1_a1, q1_a2, __ = q1.suggested_answer_ids
```
(note that your linter may not like it, but we reserve the single char `_` to translations...